Svc/Subtopologies/ComCcsds/ComCcsdsConfig/ComCcsdsConfig.fpp

```diff
@@ -13,8 +13,8 @@ module ComCcsdsConfig {
     }
 
     module Priorities {
-        constant comQueue   = 101
-        constant aggregator = 100
+        constant comQueue   = 24
```

Comment:
Com queue is highest priority downlink, so it should probably be at 29, with aggregator at 30 as it "consumes" from comQueue.